src/app/components/page-header/page-header.styl

```diff
@@ -3,6 +3,24 @@
 
 $-cover-buttons-height = $button-md-line-height + 20px
 
+.-index
+	.-controls, .-nav
+		padding: 0 20px
```

Comment:
This needs to use $grid-gutter-width

Reply:
changed to `padding: 0 ($grid-gutter-width / 2)`